src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -242,32 +243,31 @@ export const commandTileIndexValidate = command({
           }
           return Projection.get(epsg).boundsToGeoJsonFeature(Bounds.fromBbox(loc.bbox), {
             source: loc.source,
-            tileName: loc.tileName,
-            isDuplicate: (outputs.get(loc.tileName)?.length ?? 1) > 1,
+            tileName: loc.tileNames.join(', '),
```

Comment:
@blacha I'm not 100% sure if this is a sufficient solution with what that is being used for, or if the `tileName` attribute and `isDuplicate` flag need more attention?

Reply:
Yeah this would need a rethink when there are more than one output tiles, I think mostly just the isDuplicate, I dont think there is a good/easy way of representing arrays in geojson that shows up nicely in say qgis

Reply:
removed isDuplicate as discussed (this information can be gathered from other output files)